### ISSUE TITLE
Move all static references to the same base template

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#853]
**TL;DR**: Static files are referenced all over the place.  They should be in one base template. 

**Narrative**
Static files are reference in more then one place.  This duplication can cause issues and confusion.  Django typically has all the static files references in one base template.  In cases, such as ours, where there is an "experimenter" view, there maybe two base files. 

**Acceptance Criteria**
- [ ] All js/css files are accessible to the html.
- [ ] All js/css files that don't need to accessed by the general public aren't.

**Implementation Notes**
One base template with all js/css files.  One exp base template for the experimenter js/css files.
